packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -97,15 +97,35 @@ export interface ICreateContainerMetadata {
 	createContainerTimestamp?: number;
 }
 
+/** @see IGCMetadata.gcFeatureSupportInfo */
+export interface GcFeatureSupportDimensions {
+	/**
+	 * Gives a way for an app to disqualify old files from GC Tombstone enforcement
+	 * Provided via Container Runtime Options
+	 */
+	appTombstoneReadiness?: number;
+}
+
 export type GCVersion = number;
 export interface IGCMetadata {
 	/**
 	 * The version of the GC code that was run to generate the GC data that is written in the summary.
+	 * If the persisted value doesn't match the current value in the code, saved GC data will be discarded and regenerated from scratch.
 	 * Also, used to determine whether GC is enabled for this container or not:
 	 * - A value of 0 or undefined means GC is disabled.
 	 * - A value greater than 0 means GC is enabled.
 	 */
 	readonly gcFeature?: GCVersion;
+
+	/**
+	 * Numerical indication of feature support as of file creation time, used to determine feature availability over time.
+	 * This info may come from multiple sources (FF code, config service, app via Container Runtime Options),
+	 * and pertains to aspects of the document that may be fixed for its lifetime.
+	 *
+	 * For each dimension, if the persisted value is less than the currently provided value,
+	 * then this file does not support the corresponding feature as currently implemented.
+	 */
+	readonly gcFeatureSupportInfo?: GcFeatureSupportDimensions;
```

Comment:
```suggestion
	readonly gcFeaturesSupportInfo?: GcFeatureSupportDimensions;
```

Reply:
`features` indicating multiple feature support.

Reply:
Also the type says `dimensions` and the property says `info`. Can we make them the same?

Reply:
Ahh "gcFeature" sounds like "the GC feature", not "related features".  OK

Reply:
Going with gcFeatureMatrix